weblate/static/loader-bootstrap.js

```diff
@@ -1542,6 +1542,33 @@ $(function () {
         });
     });
 
+    /* Click to edit position inline. Disable when clicked outside or pressed ESC */
+    $('#position-input').on('click', function() {
+        $('#position-input').hide();
+        $('#position-input-editable').show();
+        $('#position-input-editable input').focus();
+        document.addEventListener('click', clickedOutsideEditableInput);
+        document.addEventListener('keyup', pressedEscape);
+    });
+    var clickedOutsideEditableInput = function(event) {
+        if (!$.contains($('#position-input-editable')[0], event.target) &&
+            event.target != $('#position-input')[0]
```

Comment:
To avoid using something like stopEventPropagation (which is something I was trying to avoid before, hence I didn't add on click) because of it's [dangers](https://css-tricks.com/dangers-stopping-event-propagation/), I have added an additional check in both the event handlers to check that the event causing the execution of the event handler isn't the same event as onclick on $('#position-input'). That way, we can avoid the event propagation in the first click that activates the input.